support-frontend/assets/pages/[countryGroupId]/lazyRouter.tsx

```diff
@@ -0,0 +1,85 @@
+import { lazy, Suspense } from 'react';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { HoldingContent } from 'components/serverSideRendered/holdingContent';
+import { parseAppConfig } from 'helpers/globalsAndSwitches/window';
+import {
+	getAbParticipations,
+	setUpTrackingAndConsents,
+} from 'helpers/page/page';
+import { renderPage } from 'helpers/rendering/render';
+import { geoIds } from 'pages/geoIdConfig';
+
+const abParticipations = getAbParticipations();
```

Comment:
Given we get the participations before creating the router maybe this could be a client side test? But this would probably affect performance

Reply:
I'm wondering if we could have done this client side - check the test then render the right router. But I'm assuming this would negate the performance benefit completely?

Reply:
Hmm possibly, we would have to move the routing logic out of this entryb point and into 2 modules eg. a `nonLazyRouter` and  a`lazyRouter` and conditionally lazy load  one of these based on the AB test participation, hard to know how that would impact performance, however this server side approach is testing something closer to what would actually be implemented vs the aforementioned client-side approach so is worth measuring.